.github/workflows/unit-tests-gh-hosted.yml

```diff
@@ -1,141 +0,0 @@
-        #     mongo-version: "4.4"
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Install Required System Packages
-        run: sudo apt-get update && sudo apt-get install libxmlsec1-dev lynx
-
-      - name: Start MongoDB
-        uses: supercharge/mongodb-github-action@1.7.0
-        with:
-          mongodb-version: ${{ matrix.mongo-version}}
-
-      - name: Setup Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Get pip cache dir
-        id: pip-cache-dir
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
-
-      - name: Cache pip dependencies
-        id: cache-dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.pip-cache-dir.outputs.dir }}
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/edx/testing.txt') }}
-          restore-keys: ${{ runner.os }}-pip-
-
-      - name: Install Required Python Dependencies
-        env:
-          PIP_SRC: ${{ runner.temp }}
-        run: |
-          make test-requirements
-          if [[ "${{ matrix.django-version }}" != "pinned" ]]; then
-            pip install "django~=${{ matrix.django-version }}.0"
-            pip check # fail if this test-reqs/Django combination is broken
-          fi
-
-      - name: Setup and run tests
-        uses: ./.github/actions/unit-tests
-
-  collect-and-verify:
-    if: (github.repository != 'openedx/edx-platform' && github.repository != 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == true))
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        python-version: [ '3.11' ]
-        django-version:
-          - "pinned"
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Install Required System Packages
-        run: sudo apt-get update && sudo apt-get install libxmlsec1-dev
-
-      - name: Setup Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Get pip cache dir
-        id: pip-cache-dir
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
-
-      - name: Cache pip dependencies
-        id: cache-dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.pip-cache-dir.outputs.dir }}
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/edx/testing.txt') }}
-          restore-keys: ${{ runner.os }}-pip-
-
-      - name: Install Required Python Dependencies
-        run: |
-          make test-requirements
-          if [[ "${{ matrix.django-version }}" != "pinned" ]]; then
-            pip install "django~=${{ matrix.django-version }}.0"
-            pip check # fail if this test-reqs/Django combination is broken
-          fi
-
-      - name: verify unit tests count
-        uses: ./.github/actions/verify-tests-count
```

Comment:
We had this in the github runners before but not in the old runners? Do we need to move this job to the other test file now that all tests are running on the GH runners?

Reply:
It's same as the one in https://github.com/openedx/edx-platform/pull/34708/files#diff-4a1c9e2d4ca32b59fb16b9e27b6b8807b700f47c3c8a5ae57ee1a8df5171216f . should we move it to `unit-tests.yml` file?

Reply:
Oh, I see, that's confusing, yes, let's move that job to the `unit-test.yml` as well.

Reply:
Done https://github.com/openedx/edx-platform/pull/34708/commits/3d37b1c2ed27a35a540f250ddf97ee5046a5a3aa